tests/sentry/ingest/ingest_consumer/test_dlq.py

```diff
@@ -66,14 +65,16 @@ def test_dlq_invalid_messages(factories, topic_name, consumer_type) -> None:
     )
     strategy = factory.create_with_partitions(Mock(), Mock())
 
-    # Valid payload raises original error
-    with pytest.raises(Exception) as exc_info:
-        message = make_message(valid_payload, partition, offset)
-        strategy.submit(message)
-    assert not isinstance(exc_info.value, InvalidMessage)
+    # Empty event raises InvalidMessage error (except for Attachments, which drops these in FilterStep)
+    if consumer_type != ConsumerType.Attachments:
```

Comment:
Temporary fix, will be addressed by https://github.com/getsentry/sentry/pull/68051